src/IROperator.h

```diff
@@ -1909,6 +1908,28 @@ Expr saturating_cast(Expr e) {
  * maximum values of the result type. */
 EXPORT Expr saturating_cast(Type t, Expr e);
 
+/** Prevents the simplifier from changing floating-point operations
+ * within the argument expression while returning the argument as its result.
+ * Can be used to ensure stricter floating-point semantics but does not by itself
+ * affect lower-level code generation. Mostly provided for debugging. */
```

Comment:
Re: "mostly" for debugging: I guarantee you we will end up with production code using and depending on this if we leave it like this.

If it's *entirely* for debugging, we should make the name reflect that, and/or always emit a warning when compiling it.

If it's not entirely for debugging, what are the legitimate scenarios we would want to support?